### PR TITLE
Catch errors properly in modal

### PR DESCRIPTION
**Pull request by emilylwan**
## Pull Request Summary
Fixed window alert to catch errors properly.

### Change types

- [x] Non-breaking change
- [x] Front end change

### Testing
- [ ] I have added testing to this PR

### CODE REVIEW

File: front-end/src/components/TranscriptUploadModal.jsx [hunk 1]
```diff
@@ -40,12 +40,11 @@ class TranscriptUploadModal extends React.Component {
...
-    const result = await TranscriptAPI.post(file);
-    if(result.status === 500){
-      window.alert("Error in uploading transcript. Please try again.")
-    }
-    else {
+    try {
+      await TranscriptAPI.post(file);
       this.props.toggleModal();
+    } catch (e) {
+      window.alert("Error in uploading transcript. Please try again.");
```
Comment (WeissSa) [inline front-end/src/components/TranscriptUploadModal.jsx:47]: Maybe display the error message. Something like

```suggestion
      window.alert(`Error in uploading transcript. ${e.message}`);
```
LMK if you think it's a good idea

**Review by WeissSa — APPROVED** [review]
👍 Code LGTM. Not currently on a computer where I can QA

File: front-end/src/components/TranscriptUploadModal.jsx (hunk 1 above)
Comment (emilylwan) [reply front-end/src/components/TranscriptUploadModal.jsx]: I did add in the error message and it's showing result returned with error 500. Not sure exactly how useful that is.

Comment (WeissSa) [reply front-end/src/components/TranscriptUploadModal.jsx]: Fair enough. I'm fine with either version.

**Review by helias22 — APPROVED** [review]

**Pull request merged**